compat.js

```diff
@@ -0,0 +1,15 @@
+const MAC_COMPAT_KEY = Symbol('macCompat')
+export const MAC_COMPAT = { [MAC_COMPAT_KEY]: true }
+
+export function applyCompat(code, window, overrides, dir = 'forward') {
```

Comment:
Since we are fighting for every bite of the library, let’s remove `dir` argument if we are not using it right now.

Reply:
I don't really like `dir` as well, but we need some kind of indication of the direction we're transforming in: either one way for hotkeys (meta -> ctrl), or in the reverse direction for hints (ctrl -> meta). I removed the default parameter value though.